axum/CHANGELOG.md

```diff
@@ -7,11 +7,417 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 # Unreleased
 
+- None.
+
+# 0.6.0 (25. November, 2022)
+
+## Routing
+
+- **fixed:** Nested routers will now inherit fallbacks from outer routers
+  unless they define their own fallback ([#1521])
```

Comment:
This is just compared to earlier RCs, I wonder if we should have that in its own section 🤔